test/fixtures/test-runner/output/test-timeout-flag.js

```diff
@@ -0,0 +1,16 @@
+// Flags: --test-timeout=500
+'use strict';
+const { describe, test } = require('node:test');
+const { setTimeout } = require('node:timers/promises');
+
+describe('--test-timeout is set to 500ms', () => {
+  test('test 1 must fail due to testTimeoutFailure with error test timed out after 500ms', async () => {
+      await setTimeout(1000);
+  })
+  test('test 2  must fail due to testTimeoutFailure with error test timed out after 600ms', {timeout: 600}, async () => {
+      await setTimeout(1000);
+  })
+  test('test 3 must pass', async () => {
+        await setTimeout(10);
+  })
+})
```

Comment:
```suggestion
});

```

Reply:
Thanks! I was wondering why `make lint-js-fix` wouldn't be able to catch it 😢 is it because it is a new file?

Reply:
No, it's because it's in the `fixtures` folder – maybe we should reenable it for `output` folders, but currently it has to be manual

Reply:
ohh I see, thanks!